src/dlr.cc

```diff
@@ -281,6 +300,71 @@ extern "C" int CreateDLRModel(DLRModelHandle* handle, const char* model_path,
   API_END();
 }
 
+/*! \brief Translate c args from ctypes to std types for DLRModel ctor.
+ */
+extern "C" int CreateDLRModelFromPaths(DLRModelHandle* handle, const DLRPaths* paths,
+                                       int dev_type, int dev_id) {
+  API_BEGIN();
+  DLContext ctx;
+  ctx.device_type = static_cast<DLDeviceType>(dev_type);
+  ctx.device_id = dev_id;
+
+  ModelPath path;
+  path.model_lib = paths->model_lib;
+  path.params = paths->params;
+  path.model_json = paths->model_json;
+  path.ver_json = paths->ver_json;
+  path.metadata = paths->metadata;
+  path.relay_executable = paths->relay_executable;
+  
+  DLRModel* model;
+  try {
+    if (paths->model_lib != NULL and paths->params != NULL and paths->model_json != NULL) {
```

Comment:
Is `and` a C/C++ keyword? Is it identical to `&&` ?

Reply:
doh....was clearly writing python just before this................